tests/unit/api/observation_table/test_utils.py

```diff
@@ -0,0 +1,50 @@
+"""
+Test observation table utils
+"""
+import textwrap
+
+from featurebyte import get_version
+from featurebyte.api.obs_table.utils import get_definition_for_obs_table_creation_from_view
+
+
+def test_get_definition_for_obs_table_creation_from_view(catalog, snowflake_event_table):
+    """
+    Test get_definition_for_obs_table_creation_from_view
+    """
+    _ = catalog
+    view = snowflake_event_table.get_view()
+    pruned_graph, mapped_node = view.extract_pruned_graph_and_node()
+    definition = get_definition_for_obs_table_creation_from_view(
+        graph=pruned_graph,
+        node=mapped_node,
+        name="test_obs_table",
+        sample_rows=100,
+        columns=["event_id", "event_name"],
+        columns_rename_mapping={"event_id": "id"},
+        context_name="test_context",
+        skip_entity_validation_checks=True,
+        primary_entities=["event_id"],
+    )
+    version = get_version()
+    expected_definition = f"""
+    # Generated by SDK version: {version}
+    from bson import ObjectId
+    from featurebyte import EventTable
+
+    event_table = EventTable.get_by_id(ObjectId("6337f9651050ee7d5980660d"))
+    event_view = event_table.get_view(
+        view_mode="manual",
+        drop_column_names=["created_at"],
+        column_cleaning_operations=[],
+    )
+    output = event_view.create_observation_table(
+        name="test_obs_table",
+        sample_rows=100,
+        columns=["event_id", "event_name"],
+        columns_rename_mapping={{"event_id": "id"}},
```

Comment:
Is the double curly brackets correct?

Reply:
these are doubled here as escape characters